cli/ops/process.rs

```diff
@@ -73,40 +83,84 @@ fn op_run(
   // TODO: make this work with other resources, eg. sockets
   let stdin_rid = run_args.stdin_rid;
   if stdin_rid > 0 {
-    c.stdin(resources::get_file(stdin_rid)?);
+    let file = (CloneFileFuture { rid: stdin_rid }).wait()?.into_std();
```

Comment:
this is blocking? (Hopefully will be nicer in tokio 0.2.)
https://docs.rs/futures/0.1.29/futures/future/trait.Future.html#method.wait

Reply:
It is, there's no method to clone `tokio::fs::File` synchronously so I just blocked for this promise to resolve. It is only used in this place in case you want to redirect stdio of child to file.